products/compute/terraform.yaml

```diff
@@ -470,7 +470,41 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       post_create: templates/terraform/post_create/labels.erb
   GlobalForwardingRule: !ruby/object:Overrides::Terraform::ResourceOverride
-    exclude: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "global_forwarding_rule_http"
+        primary_resource_id: "default"
+        vars:
+          forwarding_rule_name: "global-rule"
+          http_proxy_name: "target-proxy"
+          backend_service_name: "backend"
+    properties:
+      backendService: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      creationTimestamp: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      IPAddress: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      IPProtocol: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: 'caseDiffSuppress'
+        default_from_api: true
+      loadBalancingScheme: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      network: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      portRange: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: 'portRangeDiffSuppress'
+      ports: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      subnetwork: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      target: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: 'compareSelfLinkRelativePaths'
+        custom_expand: 'templates/terraform/custom_expand/self_link_from_name.erb'
```

Comment:
It looks like in the original resource we used `d.Get('target')`, so normal `ResourceRef` behaviour should be compatible. Was it your experience that we needed to explicitly set both of these?

Reply:
nope! I can remove it